src/Backend/Modules/ContentBlocks/Command/CopyContentBlocksToOtherLocaleHandler.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\Command;
+
+use Backend\Core\Engine\Model;
+use Backend\Modules\ContentBlocks\Entity\ContentBlock;
+use Backend\Modules\ContentBlocks\Repository\ContentBlockRepository;
+
+final class CopyContentBlocksToOtherLocaleHandler
```

Comment:
I only added events to replace the current events,
this is only triggered when copying the pages from one language to an other, so I didn't create a new event for it